docs/.vuepress/components/cdr-doc-code-snippet.vue

```diff
@@ -9,23 +11,32 @@
         <button class="cdr-doc-code-snippet__copy-action">
           <img class="cdr-doc-code-snippet__action-icon" :src="$withBase('/Copy@2x.png')" alt="Copy to clipboard"/>
         </button>
-        <div class="cdr-doc-code-snippet__notification" aria-live="polite">
-          <span class="cdr-doc-code-snippet__notification-message" v-if="copied">
+        <span class="cdr-doc-code-snippet__tooltip cdr-doc-code-snippet__tooltip--show-on-hover">
+          Copy to clipboard
+        </span>
+        <span class="cdr-doc-code-snippet__tooltip cdr-doc-code-snippet__tooltip--copied-notification" aria-live="polite">
+          <span class="cdr-doc-code-snippet__tooltip-message" v-if="copied">
             Copied!
           </span>
-          <span class="cdr-doc-code-snippet__notification-message" v-if="copyError">
+          <span class="cdr-doc-code-snippet__tooltip-message" v-if="copyError">
             Could not copy to clipboard.
           </span>
-          <span class="cdr-doc-code-snippet__notification-message" v-if="copyNotSupported">
+          <span class="cdr-doc-code-snippet__tooltip-message" v-if="copyNotSupported">
             This browser does not support automatic copying to clipboard.
           </span>
-        </div>
+        </span>
       </div>
       <a class="cdr-doc-code-snippet__action" :href="repositoryHref" v-if="repositoryHref">
         <img class="cdr-doc-code-snippet__action-icon" :src="$withBase('/Github@2x.png')" alt="View source in repository"/>
+        <span class="cdr-doc-code-snippet__tooltip cdr-doc-code-snippet__tooltip--show-on-hover">
```

Comment:
it would be nice to have some hover intent here to only show the tool tip after a half/sec or so?

Reply:
@mhewson I've added a `.5s` delay to the hover animation